swap/src/protocol/bob/event_loop.rs

```diff
@@ -88,6 +88,7 @@ impl EventLoop {
         let _ = Swarm::dial(&mut self.swarm, &self.alice_peer_id);
 
         loop {
+            // Note: We are making very elaborate use of `select!` macro's feature here. Make sure to read the documentation thoroughly: https://docs.rs/tokio/1.4.0/tokio/macro.select.html
```

Comment:
```suggestion
            // Note: We are making very elaborate use of `select!` macro's precondition feature here. Make sure to read the documentation thoroughly: https://docs.rs/tokio/1.4.0/tokio/macro.select.html
```

Reply:
Actually, I meant to say _features_ here. Will fix!

Reply:
Might do in a follow-up, don't want to wait for too many builds again :)